src/libsystemd/sd-netlink/rtnl-message.c

```diff
@@ -111,6 +111,20 @@ int sd_rtnl_message_route_get_flags(sd_netlink_message *m, unsigned *flags) {
         return 0;
 }
 
+int sd_rtnl_message_route_set_table(sd_netlink_message *m, unsigned char table) {
```

Comment:
If it is (or will be in the future) possible to set this higher, this should have different type (int32_t ?).